setup.py

```diff
@@ -2,13 +2,13 @@
 
 setup(
     name="mz_bokeh_package",
-    version="0.17.2",
+    version="0.18.0",
     packages=["mz_bokeh_package"],
     include_package_data=True,
 
     # Requirements for the package.
     install_requires=[
-        "requests~=2.28.0",
+        "requests~=2.31.0",
```

Comment:
```suggestion
```

Reply:
Yes, we can do that. But the issue is that we are explicitly using `requests.exception.RetryError` in the [graphqlAPI](https://github.com/materialscloud/mz-bokeh-package/blob/841e3a11c9ccf35cff6302a76d5f3d2677897b36/mz_bokeh_package/utilities/graphql_api.py#L63-L67) here in mz-bokeh-package, and then rely on requests being present because we choose the requests package as an option for the `gql` package. I'm quite convinced we could remove that usage of `RetryError`, and then we would be fine with what you propose. Did you look at it? You brought it in - I believe -, and from what is written there, it sounds to me that it is necessary only in a development scenario when we might provide bad credentials. If there is something wrong with the connection in the production environment, we would see issues at completely different places, and we also might want to be informed about them in the logs or alarms. Could you check [this try-except structure](https://github.com/materialscloud/mz-bokeh-package/blob/841e3a11c9ccf35cff6302a76d5f3d2677897b36/mz_bokeh_package/utilities/graphql_api.py#L63-L67) and try to remember what you wrote it for?

Reply:
The `ReplyError` is raised by repetitive bad responses due to an incorrect API key. We can keep the upgraded requests library while downgrading the `urllib3` as you did on the parallel [PR](https://github.com/materialscloud/analysis-apps/pull/77/commits/294e8f96fba7808d4e4a310d223c134657acd567).